### ISSUE TITLE
add features

### ISSUE THREAD

**Issue by dsolissilva** [vitalidze/traccar-web#492]
Sorry for my English. 

could add the option of creating sensors report . 
Example: " Sometimes a panic button is pressed " or " higher temperature to 10 degrees "

**Comment by clinicamatiz** [comment]
Yes. It would be excellent to send an email when an event is generated by sensor. Door alarm, off vehicle, ...
